package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@moxy/next-intl",
+  "version": "0.0.0",
+  "description": "Library to integrate react-intl with Next.js, making it easy to manage the current locale based on configurable policies",
+  "main": "es/index.js",
+  "module": "es/index.js",
+  "browser": {
+    "es/react/script.js": "./es/react/script.browser.js"
+  },
+  "files": [
+    "es"
+  ],
```

Comment:
`es` needs to be switched for `src`.

Reply:
`és` is correct since it’s the dist folder (es modules).

Reply:
(We are using Babel to transpire this package)

Reply:
Ok!

Don't forget to run the build script in the prerelease script then!

Reply:
Done, nice catch!